Snippets/Core/Core_9.1/Core_9.1.csproj

```diff
@@ -20,4 +20,4 @@
     <PackageReference Include="OpenTelemetry" Version="1.3.0" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
The changes here also don't match: https://github.com/Particular/docs.particular.net/blob/bfff555f06651bc7b8e1a72622c41ca514e1c075/Snippets/Core/Core_9.1/Core_9.1.csproj